v3/pearlboards/pearl.json

```diff
@@ -0,0 +1,118 @@
+{
+  "name": "Pearl 40% - By Koobaczech",
+  "vendorId": "0x6963",
+  "productId": "0x6965",
+  "menus": ["qmk_rgblight",
+  {
+    "label": "Audio",
+    "content": [
+      {
+        "label": "General",
+        "content": [
+          {
+            "label": "Enable Buzzer",
+            "type": "toggle",
+            "content": ["id_qmk_audio_enable", 4, 1]
+          },
+          {
+            "label": "Audio Clicky Enable",
+            "type": "toggle",
+            "content": ["id_qmk_audio_clicky_enable", 4, 2]
+          }
+        ]
+      }
+    ]
+  }
+],
```

Comment:
I don't see any audio things in your QMK config?
https://github.com/qmk/qmk_firmware/tree/master/keyboards/pearlboards/pearl

Reply:
My friend, please take a look at the Pearl rules.mk file [here](https://github.com/qmk/qmk_firmware/blob/master/keyboards/pearlboards/pearl/rules.mk), and take mention of where it says 
**AUDIO_ENABLE = yes          # Audio output**

My best, Koob